### DISCUSSION TITLE
Incrementally migrating from SwiftUI to Jetbrains Compose

### DISCUSSION THREAD

**Discussion by azlekov** [arkivanov/decompose-multiplatform-template#7] [Q&A]
First of all, great sample and template. 

I'm wondering if it is possible to mix both SwiftUI and Jetbrains Compose in a single app. My idea is if a have an app written using decompose + Jetpack Compose and Swift UI, incrementally to start switching screens to Jetbrains Compose. 

Maybe a very small example in this template or snippet here as a guideline would be greatly appreciate!

Thanks!

**Comment by arkivanov** [comment]
Thanks for the question! It's absolutely possible and there is already an example for this. Take a look at [Confietti](https://github.com/joreilly/Confetti), its SessionDetailsUi [delegates](https://github.com/joreilly/Confetti/blob/54fcf27f4a72ad52ec0a0ebbc843940a4df9a5c1/iosApp/iosApp/Sessions/SessionDetailsView.swift#L39) to Compose. Hope this helps.

**Reply by arkivanov** [reply]
cc: @joreilly
